client-test-apps/js/api-model-relationship-app/src/__tests__/utils/amplifyCLI.ts

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable testing-library/await-async-utils */
+import path from 'path';
+import { EOL } from 'os';
+import * as fs from 'fs-extra';
+import _ from 'lodash';
+import { spawn } from './execUtils';
+
+const pushTimeoutMS = 1000 * 60 * 10; // 10 minutes;
+
+const defaultSettings = {
+  name: EOL,
+  envName: 'integtest',
+  editor: EOL,
+  appType: EOL,
+  framework: EOL,
+  srcDir: EOL,
+  distDir: EOL,
+  buildCmd: EOL,
+  startCmd: EOL,
+  useProfile: EOL,
+  profileName: EOL,
+  region: process.env.CLI_REGION,
+  local: false,
+  disableAmplifyAppCreation: true,
+  disableCIDetection: false,
+  providerConfig: undefined,
+  permissionsBoundaryArn: undefined,
+};
+
+export const getCLIPath = (testingWithLatestCodebase = false) => {
```

Comment:
I see that `testingWithLatestCodebase` is always `false`, so we're basically either using the path from the environment variables or the globally installed `amplify`.  Just a question, do we have this set in the env variable? or we could also just always use the local binary after `setup-dev`.

Reply:
Yeah, I intend for this to get re-merged back into the core amplify-core-e2e stuff, so this WILL be used then I think. I'll need to revisit this in order to get the `amplify-dev` stuff working, and potentially the circle CI integration.